websub-native/src/main/java/org/ballerinalang/net/websub/WebSubEmptyCallableUnitCallback.java

```diff
@@ -33,6 +34,8 @@ public void notifySuccess() {
 
     @Override
     public void notifyFailure(io.ballerina.runtime.api.values.BError error) {
-        ErrorHandlerUtils.printError("error: " + error.getPrintableStackTrace());
```

Comment:
why we can't use error.printStackTrace() ?

Reply:
I have used the `error.printStackTrace() `. But for that, we have to pass a `PrintWriter` right?

Reply:
error.printStackTrace() should also work without printwriter